hercules/amr_wind_standin.py

```diff
@@ -322,9 +322,17 @@ def process_subscription_messages(self, msg):
         pass
 
 
-def launch_amr_wind_standin(amr_input_file, amr_standin_data_file=None):
+def launch_amr_wind_standin(amr_input_file, amr_standin_data_file=None, helics_port=None):
     temp = read_amr_wind_input(amr_input_file)
 
+    # Check that helics_port is an integer
+    # If helics_port provided update with value
+    if helics_port is not None:
```

Comment:
Do we need an `else` clause? Is the idea that `helics_port` can be provided by the user as an argument, or it can come from the input yaml?

Reply:
as written if helics_port is not provided, it doesn't over write the value already implied by the input yaml, but if there is no value there we'd need to do something, not sure which way we wanted to go